main.js

```diff
@@ -0,0 +1,64 @@
+/* eslint-disable no-unused-vars */
+
+// Inititalize the seconds to zero and other variables here
+var timer = {
```

Comment:
You should only store the timer value as seconds. You can build in support to render it in various ways, but the data should be in seconds.